lua/xcodebuild/project/manager.lua

```diff
@@ -388,20 +465,21 @@ function M.rename_current_file()
   end
 
   local newFilePath = vim.fn.expand("%:p:h") .. "/" .. newFilename
-  run_rename_file(oldFilePath, newFilePath)
+  run_rename_file(oldFilePath, newFilePath, false)
```

Comment:
I see that the config option is used for some functions and for others it's not. What's the difference between them?

Reply:
That's what I mean about the config being integration specific. This and some other functions are not called from integration files, they are called directly through commands like `XcodebuildRenameCurrentFile`. That makes it impossible to determine which configuration's config we should use, and it would feel weird to just use one of the integrations for that. So for those files I hardcoded false

Reply:
Oh, I get it now. Yeah, probably it could be a part of refactoring in the future to extract those settings to an integration-independent place.